styles/components/content-media.scss

```diff
@@ -213,7 +213,7 @@ Styleguide #{$sgi-content-toolbar}
 
 //toolbar
 .content-toolbar {
-    padding: 10px;
+    padding: $gap * 2 $gap;
```

Comment:
Is this correct?

Reply:
Yes. 
It's an update from empleo, which now is `padding: $padding $padding/2;`